Source/WebCore/rendering/GridTrackSizingAlgorithm.cpp

```diff
@@ -1570,6 +1572,19 @@ void GridTrackSizingAlgorithm::setup(GridTrackSizingDirection direction, unsigne
     m_hasPercentSizedRowsIndefiniteHeight = false;
     m_hasFlexibleMaxTrackBreadth = false;
 
+    auto resolveAndSetNonAutoRowStartMarginsOnRowSubgrids = [&] {
+        if (m_direction == ForRows) {
```

Comment:
This condition is enforced by the caller.

Reply:
Fixed